### PR TITLE
[KOGITO-1240] Add command line completion to the Kogito CLI

### PR DESCRIPTION
**Pull request by errantepiphany**
[KOGITO-1240] Add command line completion to the Kogito CLI
https://issues.redhat.com/browse/KOGITO-1240

Signed-off-by: David Ward <dward@redhat.com>

Many thanks for submiting your Pull Request :heart:! 

Please make sure that your PR meets the following requirements:

- [ ] You have read the [contributors guide](CONTRIBUTING.MD#sending-a-pull-request)
- [ ] Pull Request title is properly formatted: `[KOGITO-XYZ] Subject`
- [ ] Pull Request contains link to the JIRA issue
- [ ] Pull Request contains description of the issue
- [ ] Pull Request does not include fixes for issues other than the main ticket
- [ ] Your feature/bug fix has a unit test that verifies it
- [ ] You've tested the new feature/bug fix in an actual OpenShift cluster

### CODE REVIEW

File: cmd/kogito/command/completion/completion.go [hunk 1]
```diff
@@ -0,0 +1,103 @@
+// Copyright 2019 Red Hat, Inc. and/or its affiliates
```
[the remark this answers is not in the document]
Comment (ricardozanini) [reply cmd/kogito/command/completion/completion.go]: @errantepiphany you don't need to add the year manually, there's a script for it once you run `make test`

File: cmd/kogito/command/completion/completion.go [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+        },
+    }
+}
+
+func (i *completionCommand) InitHook() {
+    i.Parent.AddCommand(i.command)
+}
+
+func (i *completionCommand) Exec(cmd *cobra.Command, args []string) error {
+    if len(args) != 1 {
```
[the remark this answers is not in the document]
Comment (ricardozanini) [reply cmd/kogito/command/completion/completion.go]: It is. We don't need to validate args on `Exec`.

File: cmd/kogito/command/completion/completion.go (hunk 1 above)
[the remark this answers is not in the document]
Comment (errantepiphany) [reply cmd/kogito/command/completion/completion.go]: @radtriste , @ricardozanini : Every other go file has the Copyright year hardcoded. How is this different (other than being the wrong year)?

[the remark this answers is not in the document]
Comment (radtriste) [reply cmd/kogito/command/completion/completion.go]: it was just a matter of the year as you just created the file...

[the remark this answers is not in the document]
Comment (errantepiphany) [reply cmd/kogito/command/completion/completion.go]: Ah okay, thank you. :)

**Comment by ricardozanini** [comment]
@radtriste were you able to build it locally with this change on the `go.mod`?

**Comment by radtriste** [comment]
I did not get any problem when building

**Pull request merged**